src/Sulu/Bundle/ContentBundle/Content/Structure/ExcerptStructureExtension.php

```diff
@@ -90,7 +90,10 @@ public function __construct(
      */
     public function save(NodeInterface $node, $data, $webspaceKey, $languageCode)
     {
-        foreach ($this->excerptStructure->getProperties() as $property) {
+        $excerptStructure = $this->getExcerptStructure();
```

Comment:
yes we should rethink the extension stuff that this should not use the template stuff! maybe we should implement a form xml structure like @dantleech and i thought about! which would be much easier to use!